### PR TITLE
Fixing font name on DC widget converter

### PR DESCRIPTION
**Pull request by joaquinrivero**
Resolves: [MWPW-158302](https://jira.corp.adobe.com/browse/MWPW-158302)

Test URL:
Before: https://www.adobe.com/acrobat/online/compress-pdf.html
After:
https://heading-font--dc--adobecom.hlx.page/acrobat/online/sign-pdf

### CODE REVIEW

File: acrobat/styles/styles.css [hunk 1]
```diff
@@ -60,6 +60,12 @@ div [class *= PdfnowLifecycle__dropzone2___] {
   z-index: 1;
 }
 
+#CID h1, #CID [class^="DropZoneText__dropzoneDescription"],
+#CID [class^="DropZoneFooter__signInText"],
+#CID [class^="LifecycleDropZone__logoText"] {
+  font-family: 'Adobe Clean', adobe-clean,'Source Sans Pro',-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,sans-serif;
```
Comment (peller) [inline acrobat/styles/styles.css:66]: can we fix the font-family globally and not just on individual elements?

Comment (joaquinrivero) [reply acrobat/styles/styles.css:66]: I guess better idea is added to the dc-core.css, what do you think?

.FontFamily {
    font-synthesis: weight;
    font-family:  **'Adobe Clean',** adobe-clean,Source Sans Pro,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Ubuntu,Trebuchet MS,Lucida Grande,sans-serif
}

Comment (peller) [reply acrobat/styles/styles.css:66]: And that's coming from React Spectrum, so it's not something we can change easily.  Is it possible to get typekit to recognize both?

Comment (joaquinrivero) [reply acrobat/styles/styles.css]: Try many ways, this one is the more general that I found to override it

```
[class^="LifecycleDropZone"],
[class^="DropzoneContent"],
[class^="DropZoneFooter"] {
  font-family: 'Adobe Clean', adobe-clean , 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
}
```

Comment (peller) [reply acrobat/styles/styles.css]: How about `#CID *`

Not sure which is more expensive.

Comment (joaquinrivero) [reply acrobat/styles/styles.css]: Seems like targeting specific tags (e.g., #CID div, p) is more efficient

**Review by saragajic11 — APPROVED** [review]

**Review by st-angelo-adobe — APPROVED** [review]

**Review by peller — APPROVED** [review]

**Comment by Blainegunn** [comment]
please add a JIRA @joaquinrivero

**Pull request merged**